zubhub_frontend/zubhub/src/assets/js/styles/views/project_details/projectDetailsStyles.js

```diff
@@ -347,57 +389,81 @@ export const sliderSettings = images_num => ({
 });
 
 export function NextArrow(props) {
-  const { className, style, onClick } = props;
+  const { className, onClick } = props;
+  const nextArrowStyle = {
+    display: 'flex',
+    height: '200px',
+    width: '30px',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: '0 15px 15px 0',
+    background: 'var(--text-color2)',
+    zIndex: '10',
+    boxShadow:
+      '0px 3px 1px -2px rgba(0,0,0,0.2), 0px 2px 2px 0px rgba(0,0,0,0.14), 0px 1px 5px 0px rgba(0,0,0,0.12)',
+    backgroundColor: 'var(--primary-color3)',
+    '&:hover': {
+      backgroundColor: 'var(--secondary-color6)',
+    },
+  }
   return (
     <div
       className={className}
-      style={{
-        ...style,
-        display: 'flex',
-        height: '200px',
-        width: '30px',
-        justifyContent: 'center',
-        alignItems: 'center',
-        borderRadius: '0 15px 15px 0',
-        background: 'var(--text-color2)',
-        zIndex: '10',
-        boxShadow:
-          '0px 3px 1px -2px rgba(0,0,0,0.2), 0px 2px 2px 0px rgba(0,0,0,0.14), 0px 1px 5px 0px rgba(0,0,0,0.12)',
-        backgroundColor: 'var(--primary-color3)',
-        '&:hover': {
-          backgroundColor: 'var(--secondary-color6)',
-        },
-      }}
+      style={nextArrowStyle}
       onClick={onClick}
     />
   );
 }
 
 export function PrevArrow(props) {
-  const { className, style, onClick } = props;
+  const { className, onClick } = props;
+  const prevArrowStyle = {
```

Comment:
I mean to move it to makeStyles, so it creates classnames

Reply:
When I added these next to other styles in this file, I could not use the className to style the component. I tried giving the className in projectDetails.jsx and projectDetailsStyles.js as well but could not achieve the expected result. Should I import makeStyles in this file to create the classnames?

Reply:
updated this part using makeStyles